pkgs/applications/virtualization/xen/generic.nix

```diff
@@ -561,10 +562,10 @@ stdenv.mkDerivation (finalAttrs: {
     '';
 
   postFixup =
-    # Fix binaries in $out/lib/xen/bin.
+    # Fix binaries in $out/libexec/xen/bin.
```

Comment:
I think this comment is a bit superfluous since it basically repeats the line after

Reply:
There are many superfluous comments in the Xen builder, and in my opinion, there aren't enough of them yet. When I picked up this package last month, it was in a very sorry state, and I had quite a bit of trouble figuring out what does what, as there were very few comments.
In the event that I meet a sudden demise in the near future, someone else will have an easier time maintaining this if every single fetcher, patch and function is explained.